controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -77,9 +77,11 @@ bool master_thread::init()
     set_server_max_connections(SOCKET_MAX_CONNECTIONS);
     set_select_timeout(SOCKETS_SELECT_TIMEOUT_MSEC);
 
-    auto new_statistics_polling_task =
-        std::make_shared<statistics_polling_task>(database, cmdu_tx, tasks);
-    tasks.add_task(new_statistics_polling_task);
+    if (database.settings_statistics_polling()) {
+        auto new_statistics_polling_task =
+            std::make_shared<statistics_polling_task>(database, cmdu_tx, tasks);
+        tasks.add_task(new_statistics_polling_task);
+    }
```

Comment:
I guess it is ok, but basically the task is checking the configuration internally and shut down if it is not enabled.

Reply:
not needed, removed